Java/core/src/main/java/com/amazon/randomcutforest/tree/AbstractCompactRandomCutTree.java

```diff
@@ -554,6 +573,11 @@ public T maxSize(int maxSize) {
             return (T) this;
         }
 
+        public T float32Precision(boolean precision) {
```

Comment:
After we change this field to Precision, make this protected since it should be set by child classes and not by users.

Reply:
Done